<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SceneFilePage</class>
 <widget class="QWidget" name="SceneFilePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Scene Filename</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="m_sceneFile">
     <property name="toolTip">
      <string>The editor's scene configuration file. Must be specified relative to the Scene Path</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Root Scene Graph File</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="m_sceneGraphFile">
     <property name="toolTip">
      <string>The Horde3D root scene graph file. Must be relative to the scene path. If name contains a directory, it will be created if it does not already exist.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Scene Path</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="m_scenePath">
     <property name="toolTip">
      <string>Directory where all scene data will be stored</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QToolButton" name="m_setScenePath">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
